.github/workflows/PR.yml

```diff
@@ -68,3 +69,13 @@ jobs:
       macos: ${{ needs.status.outputs.macos == 'true' }}
       win-native: ${{ needs.status.outputs.win-native == 'true' }}
       release: ${{ needs.status.outputs.release == 'true' }}
+
+  validator:
+    needs: status
+    if: ${{ needs.status.outputs.validator == 'true' }}
+    uses: ./.github/workflows/validation.yml
+    secrets:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      VARA_VALIDATOR_8: ${{ secrets.VARA_VALIDATOR_8 }}
+      SSH_VARA_USERNAME: ${{ secrets.SSH_VARA_USERNAME }}
+      VARA_SSH_PRIVATE_KEY: ${{ secrets.VARA_SSH_PRIVATE_KEY }}
```

Comment:
https://docs.github.com/en/actions/using-workflows/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow